src/main/java/com/dev/cinema/controller/AuthenticationController.java

```diff
@@ -0,0 +1,23 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.dto.UserRequestDto;
+import com.dev.cinema.service.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService service;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/register")
+    public void registration(@RequestBody UserRequestDto requestDto) {
```

Comment:
```suggestion
    public void register(@RequestBody UserRequestDto requestDto) {
```
method names should be verbs